core/arch/arm/kernel/generic_entry_a64.S

```diff
@@ -47,7 +47,8 @@
 		adr	x2, stack_tmp_stride
 		ldr	w1, [x2]
 		mul	x2, x0, x1
-		adr	x1, stack_tmp
+		adrp	x1, stack_tmp
+		add	x1, x1, :lo12:stack_tmp
```

Comment:
Oh, in addition, it makes the code non-relocatable. Again, not sure how much that matters.